### PR TITLE
Add validation testing for core-vpc json files

### PR DESCRIPTION
**Pull request by donmasters**
- json validation file created
- created script to check core-vpc json files against
  validation json file
- github actions job created to trigger on PR for specific json file
  changes

### CODE REVIEW

File: .github/workflows/validate-core-vpc-configuration.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+name: "Validate core-vpc configuration"
+
+on: 
+  pull_request:
+    branches:
+      - master
```
Comment (jakemulley) [inline .github/workflows/validate-core-vpc-configuration.yml:6]: This should be `main` 👍🏼

File: .github/workflows/validate-core-vpc-configuration.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+name: "Validate core-vpc configuration"
+
+on:
```
Comment (jakemulley) [inline .github/workflows/validate-core-vpc-configuration.yml:3]: Is it worth adding a `workflow_dispatch` trigger ([example](https://github.com/ministryofjustice/modernisation-platform/blob/main/.github/workflows/check-links.yml#L4)) so we can run this manually via GitHub actions?

**Review by jakemulley — CHANGES_REQUESTED** [review]

File: .github/workflows/validate-core-vpc-configuration.yml (hunk 2 above)
Comment (donmasters) [reply .github/workflows/validate-core-vpc-configuration.yml:3]: added

File: .github/workflows/validate-core-vpc-configuration.yml (hunk 1 above)
Comment (donmasters) [reply .github/workflows/validate-core-vpc-configuration.yml]: updated to main

**Review by jakemulley — APPROVED** [review]

**Pull request merged**